CWP/papers/HSF-CWP-2017-04_doma/latex/hsf-cwp-dm.tex

```diff
@@ -257,9 +217,9 @@ \subsection{HEP Workflow in Context}\label{hep-workflow-in-context}
   resources) and exploiting the available data bandwidth between the
   storage and the compute locations. Data replication also serves to
   guard against data loss. Data replication places modest requirements
-  on the rate of IOP transactions that need to be supported, but drives
+  on the IOPS that need to be supported, but drives
   the bandwidth requirements that are needed in enacting site-to-site
-  data transfers between WAN endpoints (e.g. 10-100~Gbps). This
+  data transfers between WAN endpoints (\emph{e.g.}, 10-100~Gbps). This
```

Comment:
You need to be consistent in italicising (or not) Latinate abbreviations. (I would suggest not, as per most modern style guides and observing that in most other places in the document you don't.)